src/test/java/gomedic/commons/core/MessagesTest.java

```diff
@@ -20,6 +23,23 @@ public void getSuggestions_misspelledCommand_correctSuggestionsReturned() {
         assertEquals(reply, Messages.getSuggestions(wrongCommand));
     }
 
+    @Test
+    public void getSuggestions_misspelledCommandTarget_correctSuggestionsReturned() {
+        String wrongCommand = "add t/patet";
+        List<String> closestWords = List.of(
+                "add t/patient    ",
+                "edit t/patient    ",
+                "add t/doctor    ",
+                "view t/patient    ",
+                "list t/patient    "
```

Comment:
Why don't the suggestion just return add t/patient here as the first word is correct already and the 2nd work should be closest to t/patient?

Reply:
Actually yeah as what said, this method is pretty dangerous because `aix t/patient` would return `list t/patient` also, which is not really that close actually but because of the `t/patient` everything becomes "close"

Reply:
Should i include a check to see if either first or second part of the input string matches any existing suggestion part

Reply:
Hem I think personally, You can separate the type and the command suggestion, so you do pattern matching for the type part and the command part, and combine them later